source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -854,13 +851,10 @@ void Graph::setConstant(UiNodePtr node, mx::InputPtr& input, const mx::UIPropert
         if (val && val->isA<int>())
         {
             int prev = val->asA<int>(), temp = val->asA<int>();
-            ImGui::SameLine();
```

Comment:
All of these "same lines" are not needed in general since items are added into table cells.